src/aapt2/aapt2.csproj

```diff
@@ -1,14 +1,13 @@
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.Build.NoTargets/2.0.1">
```

Comment:
Can you put a `global.json` in the top directly like done here:

https://github.com/xamarin/net6-samples/blob/main/global.json

```json
{
    "msbuild-sdks": {
            "Microsoft.Build.NoTargets": "2.0.1"
    }
}
```

I think that would allow us to put `<Project Sdk="Microsoft.Build.NoTargets">` in every file and omit the version number? Hopefully it works for sub-directories.

Reply:
Looks like it, much cleaner!